edweiss-firebase/functions/src/functions/lectures/createQuestion.ts

```diff
@@ -0,0 +1,27 @@
+import { Timestamp } from 'firebase-admin/firestore';
```

Comment:
It alright to use directly the firestore `Timestamp`, but remember to avoid surprises in the future, only use it on the cloud function side. Privilege on the front-end the our curtum `Timestamp` interface.

Reply:
Okay thanks for the warning